azure-pipelines.yml

```diff
@@ -18,6 +18,12 @@ steps:
 - script: npm ci
   displayName: 'Install dependencies'
 
+- script: npm run lint:fix
+  displayName: 'linting'
+
+- script: npm test -- --ci --watchAll=false --passWithNoTests
```

Comment:
maybe we want to move this task after build? I mean... if we want to test, we need a build in first place, right?

Reply:
I agree to this change

Reply:
We don't need a build to run the tests. The steps might be 
- Install dependencies. (15s)
- Lint (1s)
- Unit tests (1s)
- Build (7s)

so we won't waste time if someone has linting/test issues in his/her PR.
What do you think?

Reply:
I agree with Ruben, but Fernando point to me that maybe in the Future if we have a more complicated pipeline (with many stages) all the tests (no just unit test) will be on one stage, so that is why it is usually put this task after build, but I do not know if we have to consider this right now.

Reply:
I'm thinking more in a .net core app where a binary is needed to run some tests. However for this Node-based project we're good running the tests with Jest first